docs/evaluate/major-components.mdx

```diff
@@ -41,30 +38,29 @@ Or jump straight into the SDK docs for your preferred language:
 - [TypeScript SDK](/dev-guide/typescript)
 - [.NET SDK API reference](https://dotnet.temporal.io/)
 
-For a deep dive into how Temporal SDKs work, read the [Temporal SDKs Encylopedia page](/encyclopedia/temporal-sdks).
+For a deep dive into how Temporal SDKs work, visit the [Temporal SDKs Encyclopedia page](/encyclopedia/temporal-sdks).
 
 ## Temporal Service
 
 **What is a Temporal Service?**
 
-A Temporal Service maintains the state of your application and ensures that your application resumes when failures are detected.
+The Temporal Service maintains the state of your application and ensures it resumes when failures are detected.
 
 There are three categories of Temporal Services, all derived from the [open-source Temporal Server software](https://github.com/temporalio/temporal).
-And a Temporal Application can switch between all three.
+Temporal Applications can switch between all three.
 
-- Development Temporal Service: Available in the Temporal CLI, this service is great for local development.
-- Self-hosted Temporal Service: We call a Temporal Service "Self-hosted" when someone runs and manages the Temporal Service and all of the dependencies on their own.
-- Temporal Cloud: This Temporal Technologies' proprietary instance a of a Temporal Service, completely managed by Temporal Technologies.
+- **Development Temporal Service:** This is ideal for local development and available through the Temporal CLI.
```

Comment:
I think it's "intended" to provide a safe and reliable environment to develop your Temporal applications using a local/in-house service. It supports that development with command-line based calls that help you side-step the need to write full applications to initiate or query workflows, among other features.

Reply:
Your thoughts are accurate but I would argue the phrasing is incredibly wordy and overly complicated for the intended purpose of this page.